system/jlib/jiface.hpp

```diff
@@ -68,7 +68,7 @@ class CSimpleInterfaceOf : public INTERFACE
 {
     friend class CInterfaceOf<INTERFACE>;    // want to keep xxcount private outside this pair of classes
 public:
-    inline virtual ~CSimpleInterfaceOf() {}
+    inline virtual ~CSimpleInterfaceOf() { xxcount.store(0, std::memory_order_relaxed); }
```

Comment:
Should this set xxcount to 0 or decrement it by 1 ?

Reply:
This can be removed - it was used to make the issue more reproducible, but has no benefit with the new scheme.

Reply:
just commenting as reminder still needs removing.

Reply:
now removed